lib/src/base-styles/_nx-global-header.scss

```diff
@@ -9,11 +9,12 @@
 
 .nx-global-header {
   align-items: center;
-  border-bottom: var(--nx-border-default);
+  background-color: #FFFFFF;
   display: flex;
   grid-area: header; // see nx-page's grid-template
-  min-height: 32px;
-  padding: var(--nx-spacing-6x);
+  margin-bottom: var(--nx-spacing-6x);
+  min-height: 42px;
+  padding: 0 var(--nx-spacing-6x);
 
   &:empty {
```

Comment:
This case is missing in the mockup, but I recall @alexalyons126 saying it should just be an empty white rectangle rather than disappearing like it used to.  That should be added to the mockups

Reply:
> This case is missing in the mockup, but I recall @alexalyons126 saying it should just be an empty white rectangle rather than disappearing like it used to. That should be added to the mockups

When empty, the header is simply a white rectangle.
Check 79ce920b0c66bb1e6aa69204af90740a1c3044ca